src/nwp_consumer/cmd/main.py

```diff
@@ -160,6 +165,8 @@ def run(arguments: dict) -> int:
     if arguments['--create-latest']:
         service.CreateLatestZarr()
 
+    return 0
```

Comment:
is this right, not quite sure what its about, but looks like it could have been left in accidently

Reply:
Nothing uses that as it stands - I'm working on a change to make it return a list of paths to stored files instead of an int which will be a) more useful and b) in a different PR!